packages/contracts/src/connect-contracts.ts

```diff
@@ -0,0 +1,122 @@
+import { Signer, Contract, providers, ethers } from 'ethers'
+import { Provider } from '@ethersproject/abstract-provider'
+import { getL1ContractData, getL2ContractData } from './contract-data'
+
+export type Network = 'goerli' | 'kovan' | 'mainnet'
+interface L1Contracts {
+  addressManager: Contract
```

Comment:
I think it might be better to use the full contract names (the upper case ones), e.g., `addressManager` => `Lib_AddressManager` or `AddressManager`, since we're already using that format everywhere else.

Reply:
I actually have a pretty strong opinion about this for a few reasons:
- in javascript, capital case usually means something is a class (and/or component, like a react component), but they get returned as objects 
- the synthetix team (I think Justin) was ribbing us at one point about our naming scheme because we're mixing underscores and camel case
- the contract names are longer, so not very dev-ex friendly

...I figure if this is for external use, we should use names that make more sense in a javascript context (rather than solidity). that said, I'm fine using the contract names if everyone else prefers it.

Reply:
Sure, we can leave it at this for now. Worst case we can always use both.